iceoryx_posh/test/moduletests/test_roudi_service_registry.cpp

```diff
@@ -177,18 +177,29 @@ TEST_F(ServiceRegistry_test, SingleServiceDescriptionCanBeFoundWithWildcardSearc
     ::testing::Test::RecordProperty("TEST_ID", "a12c9d39-6379-4296-8987-df56a87169f7");
     auto result = sut.add(ServiceDescription("Foo", "Bar", "Baz"));
     ASSERT_FALSE(result.has_error());
-    sut.find(searchResults, iox::capro::Wildcard, iox::capro::Wildcard);
+    sut.find(searchResults, iox::capro::Wildcard, iox::capro::Wildcard, iox::capro::Wildcard);
 
     EXPECT_THAT(searchResults.size(), Eq(1));
     EXPECT_THAT(searchResults[0].serviceDescription, Eq(ServiceDescription("Foo", "Bar", "Baz")));
 }
 
+TEST_F(ServiceRegistry_test, SingleServiceDescriptionCanBeFoundWithEventName)
```

Comment:
I think it makes sense to have another test where we search for an event that does not exist (and expect an empty result).

Probably instance and service as well (is this done?), but not in this PR as it is concerned with events.

It is probably necessary after we e.g. change the search algorithm in the `ServiceRegistry` we probably need to evaluate whether the tests actually cover all relevant cases.

Reply:
Done for service and instance. I'll add another test where the service description searched for only differs in the event.